dashboard/app/models/unit.rb

```diff
@@ -1723,6 +1723,7 @@ def self.clear_cache
     raise "only call this in a test!" unless Rails.env.test?
     @@unit_cache = nil
     @@unit_family_cache = nil
+    @@script_level_cache = nil
```

Comment:
Since `@@script_level_cache` wasn't being cleared, the data set during the first test remained unchanged in subsequent tests:
https://github.com/code-dot-org/code-dot-org/blob/751e8529137d7b72ab4dee2dbc3321b4fd229cef/dashboard/app/models/unit.rb#L452-L459

Reply:
It worked previously because we seeded deprecated unit fixtures before enabling Unit caching with `Unit.stubs(:should_cache?).returns(true)`:
https://github.com/code-dot-org/code-dot-org/blob/751e8529137d7b72ab4dee2dbc3321b4fd229cef/dashboard/test/integration/callouts_test.rb#L3-L5

Reply:
[The error](https://github.com/code-dot-org/code-dot-org/pull/57772#pullrequestreview-1978375772) can be reproduced by running the command: `cd dashboard && RAILS_ENV=test RACK_ENV=test bundle exec rails test --seed 37263 test/integration/callouts_test.rb test/models/unit_test.rb`